docs/concepts/EvictionAndDisruptionBudget/README.md

```diff
@@ -0,0 +1,47 @@
+# Eviction & Placement Disruption Budget
+
+This document explains the concept of `Eviction` and `Placement Disruption Budget` in the context of the fleet.
+
+## Overview
+
+`Eviction` pertains to the act of removing resources from a target cluster propagated by a resource placement object from the hub cluster.
+
+The `Placement Disruption Budget` object protects against voluntary disruption, and in the case of the fleet, the only allowed voluntary disruption as of now is eviction.
+
+## ClusterResourcePlacementEviction
+
+An eviction object is used to remove resources from a member cluster once the resources have already been propagated from the hub cluster.
+
+Once the resources are successfully removed or if eviction cannot be executed, the eviction object won't be reconciled again.
```

Comment:
Just a nit: 'cause we do not actually wait for successful deletion, the wording here might needs some minor tweaks.